lib/core/src/Cardano/Pool/DB/Model.hs

```diff
@@ -146,6 +153,20 @@ mReadStakeDistribution epoch db@PoolDatabase{distributions} =
     , db
     )
 
+mPutStakePoolOwner :: PoolId -> PoolOwner -> ModelPoolOp ()
+mPutStakePoolOwner poolId poolOwner db@PoolDatabase{owners} =
+    ( Right ()
+    , db { owners = (entry:filter (/= entry) owners) }
```

Comment:
**[remark]**

Alternatively, could use a `Set` for this.

Reply:
True, fixed.